src/main/java/com/exadel/discountwebapp/location/controller/LocationController.java

```diff
@@ -29,8 +34,23 @@ public LocationResponseVO getLocationById(@PathVariable Long id) {
 
     @GetMapping("/country")
     public List<LocationResponseVO> getAllLocationsByCountry(
-            @RequestParam(name = "country") String country) {
-        return locationService.findAllByCountry(country);
+            @RequestParam(name = "countryCode") String countryCode) {
+        return locationService.findAllByCountryCode(countryCode);
+    }
```

Comment:
In the future this endpoint should be replaced with getLocation with rsql query parameter

as well as getAllLocationsByCity

Reply:
Okay. Will keep in mind this.